crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -167,6 +167,8 @@ $env.config = {
         header_on_separator: false # show header text on separator/border line
     }
 
+    error_style: "fancy"
```

Comment:
1. There should be comments showing the available options for `error_style` so people know what the options are.
2. I'm fine with `fancy` as our default, but I don't really like the term `narratable`. Maybe one of these is better. I kind of lean toward "simple". 
  - `plain`
  - `without_borders`
  - `screen_reader`
  - `simple`
  - `bare`

Reply:
The term "narratable" might seem strange (not a native speaker), but I like that it conveys the purpose of the setting: to be narratable by screen readers. We could have `simple` or similar instead, but then it should be noted in the comment that it's mean to be screen reader--friendly.

Reply:
1. I just added the comment back. It was there before but I accidentally deleted it in the last commit 😅 
2. `fancy` refers to how `miette` itself calls this style, and `narratable` follows the same "logic" (they call the report handler `NarratableReportHandler`). Agree, it sounds weird 😬 I'll set it to simple for now if that's ok? I'm not sure about how these kind of decisions should be made

Reply:
I'd sign off on it if `edhowland` agrees it's good. Since he's blind and uses nushell, I'd like to get his input before finalizing the name. I'll ask him in Discord #general.

Reply:
For those not following Discord, this is what edhowland said, ```"I read the PR  and the comments and suggestions. I like either simple or plain. Prefer plain over simple. Reason: No truly  hard and fast reason, but the 'bat' command (a  replacement for 'cat' written in Rust) also implemented the default  behaviour of  using box drawing characters  for the output for border decorations. To suppress this  they added the [-p, --plain] option. So, screen reader users already have at least that one firmly planted in their memories. But I am only 1 dude. Also: plain <=> fancy  seems like their are natural  antonyms  of each other. Thoughts?"```

So, since there is a precedent with `bat`, I move that we maintain that precedent and name the two options `fancy` and `plain` for key `error_style`.

Are there further thoughts?

Reply:
Renamed in 94295b88060393496fb776a34dfa76eb9abe5667! :)